README.md

```diff
@@ -28,3 +28,20 @@ at your option.
 Unless you explicitly state otherwise, any contribution intentionally submitted
 for inclusion in the work by you, as defined in the Apache-2.0 license, shall be
 dual licensed as above, without any additional terms or conditions.
+
+### Run integration tests
+
+```
+/path/to/repo/run_ci_tests.sh
```

Comment:
The convention for this is generally to just write `./`, since the expectation is that commands will be run in this repo anyway.